app/helpers/form_helper.py

```diff
@@ -84,7 +84,7 @@ def post_form_for_location(block_json, location, answer_store, request_form, err
 
 
 def disable_mandatory_answers(block_json):
-    for section_json in block_json['sections']:
+    for section_json in block_json.get('sections', []):
```

Comment:
if your changing this for one it makes sense to change for the subsequent for loops as well?